crates/sui-core/src/unit_tests/move_package_upgrade_tests.rs

```diff
@@ -1114,3 +1130,108 @@ async fn test_different_versions_across_calls() {
 
     assert!(effects.status.is_ok(), "{:#?}", effects.status);
 }
+
+#[tokio::test]
+async fn test_conflicting_versions_across_calls() {
+    // publishes base package at version 1
+    let mut runner = UpgradeStateRunner::new("move_upgrade/base").await;
+
+    // publish a depender package at version 1 that depends on the base package at version 1
```

Comment:
nit: `depender` -> `dependent` through-out.

```suggestion
    // publish a dependent package at version 1 that depends on the base package at version 1
```

Reply:
To my defense, the idea was to refer to the variable name, but then indeed it should be "publish depender package". In any case, changed to "a dependent package" as it's indeed the best choice!